src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -77,11 +55,13 @@ const ViewResults = () => {
         is_default: true,
       });
     };
-    getRecommendations()
+
+    getAllRecommendations()
       .catch(() => {
         setFetchError(true);
         setIsLoading(false);
       });
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [careerInterests, jobSearchIndex, productSearchIndex]);
```

Comment:
This has been refactored into the `service.js` file which is more in line with out best practice. It also cleans up this file a little bit.